lms/services/user.py

```diff
@@ -126,18 +126,10 @@ def get(self, application_instance, user_id: str) -> User:
 
     def _user_search_query(self, application_instance_id, user_id) -> Select:
         """Generate a query for searching for users."""
-
-        query = select(User)
-
-        # Normally we'd have an `if application_instance_id` here, for a proper
```

Comment:
I removed all of this. 

Yes, a lot other similar method do build a query conditionally but we can just add that here when we need it I don't think we need to explain it here.